src/config/server.rs

```diff
@@ -140,6 +140,24 @@ impl Default for Server {
             .map(|s| s.parse().expect("SERVER_THREADS was not a valid number"))
             .ok();
 
+        // Dynamically load the configuration for all the rate limiting actions. See
+        // `src/rate_limiter.rs` for their definition.
+        let mut rate_limiter = HashMap::new();
+        for action in LimitedAction::VARIANTS {
+            let env_var_key = action.env_var_key();
+            rate_limiter.insert(
+                *action,
+                RateLimiterConfig {
+                    rate: Duration::from_secs(
+                        env_optional(&format!("RATE_LIMITER_{env_var_key}_RATE_SECONDS"))
```

Comment:
it looks like previously we were using `WEB_NEW_PKG_RATE_LIMIT_RATE_MINUTES`. so I guess this is a breaking change? would've been good to mention that in the PR description 😉

Reply:
Sure, the environment variable is not in use right now though.